enums.py

```diff
@@ -0,0 +1,44 @@
+"""Module containing enumerated types"""
+# Author: Daniel Davis

```

Comment:
Please remove these 3 comments

Reply:
PEP8 requires string literals on modules - and they get used to automatically build the documentation. I don't get it? please explain.

Reply:
The line was about the author comments, it is on line 2 (not 1)